src/assets/css/site.css

```
@@ -270,22 +302,40 @@ html.js .js-fallback-notice {
+.search-box__clear:focus-visible,
+.note-finder__clear:hover,
+.note-finder__clear:focus-visible {
+  color: var(--color-heading);
+}
+
+.search-box__clear span,
+.note-finder__clear span {
+  display: block;
+  inline-size: 0.78rem;
+  block-size: 0.78rem;
+  background: currentColor;
```

Stylelint (17.9.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)